0001-typedb/01-start/04-iam-schema.md

```diff
@@ -0,0 +1,297 @@
+longTailKeywords: identity and access, identity and access management, iam schema explanation, typedb tutorial, typedb quickstart, learn typedb, sample app, sample application
+summary: Explanation of the IAM schema, widely used throughout TypeDB documentation.
+toc: false
+---
+
+# IAM schema explanation
+
+## Introduction
+
+**Identity and access management** (**IAM**) is a complex field that has seen massive growth over the past two decades 
+to meet user demand. IAM systems have become progressively more complicated to account for the variety of user 
+requirements and to ensure compatibility with a backlog of legacy systems, leading to the definition of standards by 
+governing bodies, such as [NIST](https://doi.org/10.6028/NIST.SP.800-63-3), in an attempt to unify and simplify the 
+state of IAM ecosystems, as well as to set out best practices. Modern IAM systems, such as the 
+[RSA’s Identity Governance and Administration Platform](https://www.rsa.com/solutions/identity-governance-and-administration/), 
+implement these models while providing APIs to existing IAM systems an organization might have in order to unify IAM 
+under a single management framework.
+
+<div class="note">
+[Note]
+The terminology used in the IAM [schema](03-quickstart.md#prepare-a-tql-file) is defined according to the sources 
+mentioned above and ISO/IEC 15408-1 standard within the context of an information system.
+</div>
+
+<div class="note">
+[Note]
+The Identity and Access Management database [schema](03-quickstart.md#prepare-a-tql-file) and accompanying 
+[dataset](03-quickstart.md#insert-data) described here are used in most examples throughout TypeDB documentation, 
+including the [Quickstart guide](03-quickstart.md).
+</div>
+
+## Overview
+
+As with any [TypeDB schema](../02-dev/02-schema.md), the IAM schema consists of:
+
+- Types
+  - Entity types
+  - Relation types
+  - Attribute types
+- Rules
+
+Read the schema documentation to better understand the terminology of TypeDB schemas.
+
+The main entity types in the IAM schema:
+
+- Subject — Active entity in the [system] that performs operations on objects. 
+  - User — Human or IT entity possibly interacting with the [system] from outside of the [system] boundary.
+    - Person — User with `credentials`, `email`, and `full-name` attributes.
+  - User-group — A group of users that share the same role.
+- Object — Passive entity in the [system], that contains or receives information, and upon which subjects perform
+  operations. 
+  - Resource — Anything usable or consumable in the [system].
+    - File — File in some filesystem with path [to the file] and size (in KB) attributes.
+  - Resource-collection — A collection of resources on which the same operations can be performed.
+- Action — An operation or operation set that can be performed on a specific type of object. Its name is stored in
+  the `action-name` attribute.
+
```

Comment:
"[system]" and "[to the file]" are used above. What does this formatting mean?

Reply:
IAM as in Identity and access management is a solution that usually serves as a part of access control in some system (for example, a banking client-server solution). And [system] here is whatever solution/system that is we are integrated in. So a user would be a user of the system and so on.

Reply:
I'll update the syntax here. Will probably use italics for system and usual parenthesis `()` for to the file clarification.